Signup screen added

Monte9 opened the pull request:
Feel free to leave comments.

Monte9 commented:
/cc @browniefed

browniefed commented on screens/signup/index.js:
```diff
@@ -0,0 +1,207 @@
+    left: 0,
+    top: 0,
+    width: Dimensions.get('window').width,
+    height: Dimensions.get('window').height
```
This is part has been revamped in latest code, Checkout the new code structure of the login page

browniefed commented on screens/signup/index.js:
```diff
@@ -0,0 +1,207 @@
+    height: Dimensions.get('window').height
+  },
+  headerIconView: {
+    flex: .1,
```
Should probably never do point flex values. This makes things complicated

browniefed commented on screens/signup/index.js:
```diff
@@ -0,0 +1,207 @@
+    height: 25
+  },
+  headerTitleView: {
+    flex: 0.1,
```
Same here, just remove flex property.

browniefed commented on screens/signup/index.js:
```diff
@@ -0,0 +1,207 @@
+  inputs: {
+    paddingTop: 20,
+    paddingBottom: 10,
+    flex: .40
```
Remove flex here.

browniefed commented on screens/signup/index.js:
```diff
@@ -0,0 +1,207 @@
+    flex: .40
+  },
+  footerContainer: {
+    flex: 0.15,
```
Remove flex.

browniefed commented on screens/signup/index.js:
```diff
@@ -0,0 +1,207 @@
+  signin: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    flex: 0.05
```
Remove flex.

browniefed commented:
Needs a little work on the flex.
Checkout the `Login` page I updated all the content in it.

browniefed commented:
More explanation on the flex. Most of those things probably just have defined heights, we can just let them be at their normal heights. The % flex stuff was a good way to basically do % widths, just can be confusing to new comers trying to learn flex.

Monte9 commented:
I addressed the PR comments. And thanks for the explanation on flex, but I disagree on not using flex. I see what you mean by using a fixed height and it is so, but I am using flex to place the elements where they need to be on the screen. Here are some screenshots showing the view on various phones using flex:
<img width="300" height="533" alt="screen shot 2016-11-22 at 10 13 14 pm" src="https://cloud.githubusercontent.com/assets/7840686/21249227/6c80a7ec-c2f1-11e6-9be3-ef253727f99d.png">
<img width="300" height="533" alt="screen shot 2016-11-22 at 10 13 14 pm" src="https://cloud.githubusercontent.com/assets/7840686/21249228/6c817884-c2f1-11e6-94fd-38c22aa88f28.png">
<img width="300" height="533" alt="screen shot 2016-11-22 at 10 13 14 pm" src="https://cloud.githubusercontent.com/assets/7840686/21249226/6c7f02d4-c2f1-11e6-92af-b4777cf92703.png">

And here's the screeshot without flex:
<img width="300" height="533" alt="screen shot 2016-11-22 at 10 13 14 pm" src="https://cloud.githubusercontent.com/assets/7840686/21249238/7c4dcbdc-c2f1-11e6-99f5-2420c41de669.png">

browniefed commented:
Ah sorry, I didn't mean don't use flex, I just meant don't use flex with hard to determine ratios.

Monte9 commented:
Ah that makes more sense. I updated the flex values to whole numbers.

browniefed commented:
Thanks, I'll get this merged likely tomorrow, and then fix things up to add in some navigation or something to swipe between screens.

Monte9 commented:
Sounds good. What do you plan to use for navigation? NavigationExperimental?

browniefed commented:
Likely ExNavigation. Easier to deal with.

browniefed commented:
I've cleaned up the styling if you care to look at the code now.
